src/layout/MasterLayout.cpp

```diff
@@ -1234,6 +1234,13 @@ std::any CHyprMasterLayout::layoutMessage(SLayoutMessageHeader header, std::stri
         runOrientationCycle(header, nullptr, -1);
     } else if (command == "orientationcycle") {
         runOrientationCycle(header, &vars, 1);
+    } else if (command == "mfact") {
+        if (vars.size() >= 2) {
+            for (auto& nd : m_lMasterNodesData) {
+                if (nd.isMaster)
+                    nd.percMaster = std::clamp(std::stof(vars[1]), 0.05f, 0.95f);
```

Comment:
every use of std::stof has to be put in a try{} block.

Secondly, no need to `std::stof` every time.

before the for loop:
```cpp
float ejfurngnurg = 0;
try {
   ejfurngnurg = std::stof(vars[1]);
} catch (std::exception& e) {
   Debug::log(LOG, "OOPSIE WOOPSIE LMAO XD {}", e.what());
}
```

Reply:
fixed